src/spot.sol

```diff
@@ -37,22 +37,22 @@ contract Spotter is LibNote {
 
     // --- Data ---
     struct Ilk {
-        PipLike pip;
-        uint256 mat;
+        PipLike pip;  // OSM
```

Comment:
This actually depends on the collateral. In general will be an OSM. But for example for SAI collateral it was just a simple DSValue. So I'd call it a generic `Price Feed`.